src/microbe_stage/PatchManager.cs

```diff
@@ -105,23 +103,39 @@ private void HandleChunkSpawns(BiomeConditions biome)
         }
     }
 
-    private void HandleCloudSpawns(BiomeConditions biome)
+    private void HandleCloudSpawns(BiomeConditions biome, Vector3 playerPosition)
     {
         GD.Print("Number of clouds in this patch = ", biome.Compounds.Count);
 
-        foreach (var entry in biome.Compounds)
+        spawnSystem.ClearBiomeCompounds();
```

Comment:
This shouldn't always happen here, this should only be triggered when the spawners are destroyed on patch change.

Reply:
I've heard the eventual goal is to have compound amounts and densities change over time. This would mean clearing the system here. I guess you are right though, that's currently not a feature, so it makes no sense now.